doc/reviewing_PR_guide.md

```diff
@@ -0,0 +1,89 @@
+
+Another way to think of this rule might be "Extraordinary claims require extraordinary evidence, and ordinary claims require ordinary evidence".  If you tell me there should be duct tape spawning in hardware stores, I won't need much of a source.  If you tell me that solar panels alone can give an electric car all the power it needs, I'm gonna need some very good evidence of that since nobody is making solar cars.
+
+## 7. Failing CI tests aren't related
+
+We have many CI tests that run for each PR.  If you're going to give a PR a passing review, *you must have looked at these*.  I'm lazy too, I know it's tempting to ignore them, but please don't.
+
+# Indicators to call for a second opinion
+
+These aren't part of the review checklist, but they constitute observations that would justify getting more than one person with merge permissions to weigh in on the change.  **These don't mean your PR isn't getting merged**.  They're just things we've noticed are often associated with PRs that become problems later.
+
+- PR creator is resistant to changes suggested to PR, particularly from collaborators and up
+- PRs that are part of a large, sweeping overhaul, without an issue posted or similar document discussing the design direction and plan for the overhaul
+- changes justified by what "players" need or want, without deeper rationale (contrary to popular belief, improving player experience *is* desirable, but our experience is that just 'asking people' will produce wildly different results depending on which people you're talking to, and changes justified this way rarely have much statistical rigour)
+- changes that revert previously controversial adjustments with new math; this can be a hard one to know, but for example, if someone starts bumping up bow damage based on their own research, the bar to merge is *very high*, as this is a topic that's been researched *to death* by now and it's highly unlikely - although not at all impossible - that a new contributor is aware of all the work done before.
+- changes to any political hot-button topics.  These are frequent trollbait.  Anything that touches on pride flags, trans/gay rights, immigration, and real-life political figures should be watched closely.
+- similarly, overly edgy content - especially stuff that relates to cannibalism, NPC servitude, etc should be monitored very closely.
+- topical humour, memes, pop-culture references, etc. are often added with the best of intentions but their expiry date generally runs out before the PR has even finished merging.  They should be taken with a large grain of salt, even when going into something like crazy cataclysm.
```

Comment:
```suggestion
- PR creator is resistant to changes suggested to PR, particularly from collaborators and up.
- PRs that are part of a large, sweeping overhaul, without an issue posted or similar document discussing the design direction and plan for the overhaul.
- Changes justified by what "players" need or want, without deeper rationale (contrary to popular belief, improving player experience *is* desirable, but our experience is that just 'asking people' will produce wildly different results depending on which people you're talking to, and changes justified this way rarely have much statistical rigour).
- Changes that revert previously controversial adjustments with new math; this can be a hard one to know, but for example, if someone starts bumping up bow damage based on their research, the bar to merge is *very high*, as this is a topic that's been researched *to death* by now and it's highly unlikely - although not at all impossible - that a new contributor is aware of all the work done before.
- Changes to any political hot-button topics.  These are frequent troll bait.  Anything that touches on pride flags, trans/gay rights, immigration, and real-life political figures should be watched closely.
- Similarly, overly edgy content - especially stuff that relates to cannibalism, NPC servitude, etc should be monitored very closely.
- Topical humour, memes, pop-culture references, etc. are often added with the best of intentions but their expiry date generally runs out before the PR has even finished merging.  They should be taken with a large grain of salt, even when going into something like Crazy Cataclysm.
```